scripts/examples_master.py

```diff
@@ -674,6 +674,12 @@
                     "subcategory": "Timeseries forecasting",
                     "keras_3": True,
                 },
+                {
+                    "path": "event_classification_for_payment_card_fraud_detection",
+                    "title": "Event classification for payment card fraud detection",
+                    "subcategory": "Timeseries forecasting",
```

Comment:
Isn't it rather timeseries classification?

Reply:
You are right.
*Changed to "Timeseries classification".

To be even more accurate, we could have a new category such as "Time-sequence classification" or "Event classification", since a "Timeseries" implies a uniform sampling and a univariate numerical feature.

Reply:
You can have multidimensional timeseries. Generally all sequence data where the sequence step is a time step is referred to as timeseries.